File: TransCoder_tokenized_test_set_functions/python/SIEVE_ERATOSTHENES_0N_TIME_COMPLEXITY.py
def manipulated_seive ( N ) :
    isprime [ 0 ] = isprime [ 1 ] = False
    for i in range ( 2, N ) :
        if isprime [ i ] == True :
            prime . append ( i )
            SPF [ i ] = i
        j = 0
        while ( j < len ( prime ) and i * prime [ j ] < N and prime [ j ] <= SPF [ i ] ) :
            isprime [ i * prime [ j ] ] = False
            SPF [ i * prime [ j ] ] = prime [ j ]
            j += 1
    

File: results/java_codex/SIEVE_ERATOSTHENES_0N_TIME_COMPLEXITY.java
void manipulated_seive ( int N ) {
  isprime [ 0 ] = false;
  isprime [ 1 ] = false;
  for ( int i = 2;
  i < N;
  i ++ ) {
    if ( isprime [ i ] ) {
      prime . push_back ( i );
      SPF [ i ] = i;
    }
    for ( int j = 0;
    j < prime . size ( ) && i * prime . at ( j ) < N && prime . at ( j ) <= SPF [ i ];
    j ++ ) {
      isprime [ i * prime . at ( j ) ] = false;
      SPF [ i * prime . at ( j ) ] = prime . at ( j );
    }
  }
}